google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/BigtableDataClient.java

```diff
@@ -208,7 +213,7 @@ static BigtableDataClient createWithClientContext(
    * @throws com.google.api.gax.rpc.ApiException when a serverside error occurs
    */
   public boolean exists(String tableId, String rowKey) {
-    return ApiExceptions.callAndTranslateApiException(existsAsync(tableId, rowKey));
+    return exists(tableId, rowKey, new ExistsOptions());
```

Comment:
Here and everywhere, Instead of `new ExistsOptions()`, can we have something like `ExistsOptions.create()` so it's consistent with the rest of the API? And remember to update the javadoc after you make the change. And when we want to set ExistOptions for a view id, we can do `ExistsOptions.newBuilder().setAuthorizedViewId(..).build`, or simply `ExistsOptions.createForAuthotizedView(..)`

Reply:
Done. Please take another look.